deployment/cloudwatch-agent-config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "agent": {
+    "metrics_collection_interval": 60,
+    "run_as_user": "root"
+  },
+  "logs": {
+    "logs_collected": {
+      "files": {
+        "collect_list": [
+          {
+            "file_path": "/home/ubuntu/db-service/logs/info.log*",
+            "log_group_name": "staging-dbservice",
+            "log_stream_name": "{instance_id}",
+            "retention_in_days": -1
```

Comment:
lets keep retention, maybe 30 or 60 days otherwise we'll always forget to delete

Reply:
Added `cloudwatch.tf` file